viite-UI/src/view/FormCommon.js

```diff
@@ -40,6 +40,7 @@
       var link = _.head(_.filter(links, function (l) {
         return !_.isUndefined(l.status);
       }));
+      var roadType = !_.isUndefined(link.roadTypeId) ? link.roadTypeId : '';
```

Comment:
and then in roadType to be like 
var roadType = !_.isUndefined(link.roadTypeId) ? link.roadTypeId : RoadType.Undefined.value;